library/Makefile

```diff
@@ -91,9 +104,7 @@ shared: libmbedcrypto.$(DLEXT) libmbedx509.$(DLEXT) libmbedtls.$(DLEXT)
 # tls
 libmbedtls.a: $(OBJS_TLS)
 	echo "  AR    $@"
-	$(AR) -rc $@ $(OBJS_TLS)
-	echo "  RL    $@"
-	$(AR) -s $@
+	$(AR) $@ $(OBJS_TLS)
```

Comment:
I don't like removing options\commands because of one toolchain. 
For the -rc, have it `$(RC)`, which is empty for IAR, and `-rc` for the rest.